quobyte_driver.go

```diff
@@ -72,16 +79,30 @@ func (driver quobyteDriver) Remove(request volume.Request) volume.Response {
 	return volume.Response{Err: ""}
 }
 
+func check_mount_point(mPoint string) error {
+	// We try it 5 times ~5 seconds -> move this into config?
+	max_tries := 5
+	tries := 0
+	var mount_error error
+	for tries <= max_tries {
```

Comment:
This looks like what we want - I am not familiar enough with Quobyte to know if there is any sort of SLA for newly created volumes propagating to client mountpoints. Maybe max_tries should be exposed as a flag until this is better understood? Also, if you want to get really fancy, exponential backoff wouldn't be a bad idea:
https://github.com/rancher/windows-scheduler/blob/master/metadata.go#L77-L96